doc/reference/api/design_guidelines.rst

```diff
@@ -68,13 +68,12 @@ Examples
 Conditional Data and APIs
 *************************
 
-APIs and libraries may provide features that are expensive in RAM or
-code size but are optional in the sense that some applications can be
-implemented without them.  Examples of such feature include
-:kconfig:`capturing a timestamp <CONFIG_CAN_RX_TIMESTAMP>` or
-:kconfig:`providing an alternative interface <CONFIG_SPI_ASYNC>`.  The
-developer in coordination with the community must determine whether
-enabling the features is to be controllable through a Kconfig option.
+APIs and libraries may provide features that are expensive in RAM or code size
+but are optional in the sense that some applications can be implemented without
+them.  Examples of such feature include :kconfig:`CONFIG_CAN_RX_TIMESTAMP` or
+:kconfig:`CONFIG_SPI_ASYNC`.  The developer in coordination with the community
+must determine whether enabling the features is to be controllable through a
+Kconfig option.
```

Comment:
This is only rearranging whitespace, not fixing anything as far as I can tell. This clutters the diff.

Did you mean to make this fix instead? `s/Examples of such feature/Examples of such features/`

Reply:
I just wrapped to 80cols

Reply:
Doesn't really belong in this commit then, but ok

Reply:
:kconfig:`capturing a timestamp <CONFIG_CAN_RX_TIMESTAMP>`  ->  :kconfig:`CONFIG_CAN_RX_TIMESTAMP`

Reply:
Actually removed these 2 changes, since 

```
:kconfig:`description <CONFIG_OPTION>`
```

is a supported syntax, will update the Kconfig PR to support that.